src/PassKit/PKEnums.cs

```diff
@@ -44,8 +44,10 @@ public enum PKPassLibraryAddPassesStatus : long {
 
 	[Native]
 	public enum PKPassType : ulong {
-		Barcode, Payment,
-		// Any = ~0
+		Barcode,
+		SecureElement,
+		Payment = SecureElement,
```

Comment:
should be marked as deprecated - just for symmetry with other deprecation attributes